webdriver/state/text/get_text.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import uuid
+
+from support.asserts import assert_error, assert_success
+from support.inline import inline
+
+def test_getting_text_of_a_non_existant_element_is_an_error(session):
+   session.url = inline("""<body>Hello world</body>""")
+   id = uuid.uuid4()
+
+   result = session.transport.send(
```

Comment:
`send_session_command` assumes the session succeeds, and fails the test if it doesn't. In this one case, we want to check the actual result that comes back.